app/src/main/res/values-nl/strings.xml

```diff
@@ -208,4 +208,18 @@
     <string name="post_listing_permalink_copied">Permalink gekopieerd</string>
     <string name="comment_node_block">%1$s blokkeren</string>
     <string name="comment_reply_node_copy_permalink">Permalink kopiëren</string>
+    <string name="person_profile_activity_about">About</string>
+    <string name="person_profile_activity_posts">Posts</string>
+    <string name="person_profile_activity_comments">Comments</string>
+    <string name="sorttype_active">Actief</string>
+    <string name="sorttype_hot">Populair</string>
+    <string name="sorttype_new">Nieuw</string>
+    <string name="sorttype_old">Oud</string>
+    <string name="sorttype_topday">Top van de dag</string>
+    <string name="sorttype_topweek">Top van de week</string>
+    <string name="sorttype_topmonth">Top van de maand</string>
+    <string name="sorttype_topyear">Top van het jaar</string>
```

Comment:
In the english translation you keep them short, for parity maybe its better too?
```xml
<string name="sorttype_topday">TopDag</string>
<string name="sorttype_topweek">TopWeek</string>
<string name="sorttype_topmonth">TopMaand</string>
<string name="sorttype_topyear">TopJaar</string>
<string name="sorttype_topall">TopAlles</string>
```

Reply:
Agreed. Fixed it as well.